packages/harpguru-core/src/components/covariant-menu/covariant-menu.tsx

```diff
@@ -23,11 +22,8 @@ import {
   useSetHarpStrataByRootPitch,
 } from './hooks'
 
-export const CovariantMenu = ({
-  hideLabel,
-  hideMenu,
-  openCloseTapHandler,
-}: MenuProps): React.ReactElement => {
+export const CovariantMenu = (menuProps: MenuProps): React.ReactElement => {
```

Comment:
Done all of this except for renaming the `menuProps` instance to to `props`. Since they are passed onwards in the JSX, it's useful to give them a disambiguating name.